src/components/EditNoteModal/editnotemodal.css

```diff
@@ -0,0 +1,13 @@
+.edit-notes-parent{
+    background-color: aliceblue;
+    padding: 2rem;
+    box-shadow: 0px 0px 8px #000000;
```

Comment:
you have used `black` as color value in some previous css files and here it's `#000000`. Try to keep your code consistent

Reply:
css variable naming is my vetal I have to get over that hump and start. I agree, will do this before launch.